src/simulations/collect.jl

```diff
@@ -164,6 +165,128 @@ function run!(
     return df_agent, df_model
 end
 
+"""
+    offline_run!(model, agent_step! [, model_step!], n::Integer; kwargs...)
+    offline_run!(model, agent_step!, model_step!, n::Function; kwargs...)
+
+Do the same as [`run`](@ref), but write output to a CSV file instead of collecting
+it into an in-memory dataframe.
+Useful when the amount of collected data is expected to exceed the memory available
+during execution.
+
+## Keywords
+* `adata_savefile="adata.csv"` : a file to write agent data on.
+* `mdata_savefile="mdata.csv"`: a file to write the model data on.
+* `writing_interval=1` : write to file every `writing_interval` times data collection \
+is triggered. If the `when` keyword is not set, this corresponds to writing to file \
+every `writing_interval` steps; otherwise, the data will be written every \
+`writing_interval` times the `when` condition is satisfied \
+(the same applies to `when_model`).
```

Comment:
Please remove the ending `\` and use 2 spaces of intentation for the next row as is done in all other docstrings.